georchestra-integration/georchestra-authnz/src/main/resources/config-spring-geonetwork.xml

```diff
@@ -10,4 +10,8 @@
   <ctx:annotation-config />
   <!-- scan only the @Configuration class and let it do its job -->
   <ctx:component-scan base-package="org.georchestra.geonetwork.security.integration"/>
+
+  <bean id="ServiceManager" class="jeeves.server.dispatchers.ServiceManager" lazy-init="true"/>
+  <bean id="resources" class="org.fao.geonet.resources.FileResources"/>
+  <bean id="GeonetworkDataDirectory" class="org.fao.geonet.kernel.GeonetworkDataDirectory" lazy-init="true"/>
```

Comment:
Same remark as above (onto the spring xml config file from test, why not, but here ?)